src/components/Forum/ForumPost/PostCard.js

```diff
@@ -43,6 +59,8 @@ const PostCard = ({
     content: content,
     postId: id,
     forumId: forumId,
+    timestamp: timestamp,
```

Comment:
Because we send this data (which has a date object) to another screen using navigation, it will throw an error

`Non-serializable values were found in the navigation state.`

I guess we can suppress this warning somewhere. (If you have better alternatives, you can do that)